app/views/photos/index.html.haml

```diff
@@ -0,0 +1,2 @@
+No photos yet!
+%a(href='#') Upload a photo
```

Comment:
Prefer the hash attributes syntax (%tag{ lang: 'en' }) over HTML attributes syntax (%tag(lang=en))